src/sgame/sg_bot_parse.cpp

```diff
@@ -1276,6 +1276,7 @@ AIBehaviorTree_t *ReadBehaviorTree( const char *name, AITreeList_t *list )
 	D( UP_RADAR );
 	D( UP_JETPACK );
 	D( UP_GRENADE );
+	D( UP_FIREBOMB );
```

Comment:
This only allows to use the symbol. Technically, they still can't buy it nor throw it.

Reply:
you mean they won't want to do it, right?